canvas_modules/common-canvas/__tests__/common-properties/components/table-buttons-test.js

```diff
@@ -214,34 +233,34 @@ describe("Table buttons renders correctly", () => {
 				"tooltip": "Custom label button with Carbon icon"
 			}
 		];
-
-		const instance = wrapper.find("TableButtons").instance();
-		const actual = instance.convertToolbarConfig(STATES.DISABLED, customButtons);
-		expect(actual).to.eql(expected);
+		expect(convertToolbarConfig.calledOnce).to.equal(true);
+		expect(convertToolbarConfig.calledWith(STATES.DISABLED, customButtons)).to.equal(true);
+		expect(convertToolbarConfig.returnValues[0]).to.deep.equal(expected);
 	});
 
 	it("customButtonIconCallback calls buttonIconHandler with correct data", () => {
 		const buttonIconHandler = sinon.spy();
 		controller.setHandlers({
 			buttonIconHandler: buttonIconHandler
 		});
-		wrapper = mountWithIntl(
+		wrapper = renderWithIntl(
 			<TableButtons
 				controller={controller}
 				propertyId={propertyId}
-				customButtons={customButtons}
+				customButtons={customButtons1}
 				tableState=""
 				customButtonsState={customButtonsState}
 			/>
 		);
 
-		const instance = wrapper.find("TableButtons").instance();
-		instance.customButtonIconCallback(customButtons[2].id, customButtons[2].carbonIcon);
+		const carbonButton = wrapper.getByText("Label icon");
```

Comment:
When I use `console.log(prettyDOM(carbonButton));` here, I get this, this is not a button -
```
<div class="toolbar-icon-label before">
      Label icon
</div>
```


Use `const carbonButton = wrapper.getByRole("button", { name: /Label icon/i });` here.